src/initialize.cc

```diff
@@ -6,10 +6,17 @@
 #include <dmlc/logging.h>
 
 namespace mxnet {
+namespace op {
+void RegisterLegacyOpProp();
+void RegisterLegacyNDFunc();
+}
+
 class LibraryInitializer {
  public:
   LibraryInitializer() {
     dmlc::InitLogging("mxnet");
+    mxnet::op::RegisterLegacyOpProp();
```

Comment:
How this get invoked? Note this might depend on initationalization order. The Legacy op are not guaranteed to be initialized before this  if you use static variable